fuse2.yaml

```diff
@@ -0,0 +1,81 @@
+package:
+  name: fuse2
+  version: 2.9.9
+  epoch: 0
+  description: A library that makes it possible to implement a filesystem in a userspace program.
+  copyright:
+    - license: GPL-2.0-only AND LGPL-2.1-only
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - fuse-common
+      - gettext-dev
+      - libtool
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/libfuse/libfuse.git
+      tag: fuse-${{package.version}}
+      expected-commit: d04687923194d906fe5ad82dcd546c9807bf15b6
+
+  - uses: patch
+    with:
+      patches: fuse-namespace-conflict-fix.patch
+
+  - uses: patch
+    with:
+      patches: fuse-closefrom.patch
+
+  - runs: |
+      ./makeconf.sh
+
+  - runs: |
+      UDEV_RULES_PATH=/lib/udev/rules.d ./configure \
+        --prefix=/usr \
+        --enable-static \
+        --enable-shared \
+        --disable-example \
+        --enable-lib \
+        --enable-util \
+        --bindir=/bin
+
+  - runs: make -j$(nproc)
+
+  - uses: autoconf/make-install
+
+  - runs: rm -r "${{targets.contextdir}}"/dev "${{targets.contextdir}}"/etc/init.d
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-dev
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - fuse-common
+    description: fuse dev
+
+  - name: ${{package.name}}-doc
+    pipeline:
+      - uses: split/manpages
+    description: fuse manpages
+
+  - name: ${{package.name}}-static
+    pipeline:
+      - uses: split/static
+    description: fuse static
```

Comment:
Sadly ordering matters. `split/static` must come before `split/dev` as both split the static libraries.
In this lexical ordering -dev package ends up having static libraries and static package is empty.

```
$ tar -tf fuse2-dev-*.apk 2>/dev/null | grep usr/lib
usr/lib
usr/lib/libfuse.a
usr/lib/libfuse.so
usr/lib/libulockmgr.a
usr/lib/libulockmgr.so
usr/lib/pkgconfig
usr/lib/pkgconfig/fuse.pc

$ tar -tf fuse2-static-*.apk 2>/dev/null
.SIGN.RSA.local-melange.rsa.pub
.PKGINFO
var
var/lib
var/lib/db
var/lib/db/sbom
var/lib/db/sbom/fuse2-static-2.9.9-r0.spdx.json
```